cmd/crc/cmd/start.go

```diff
@@ -39,7 +39,7 @@ func init() {
 	flagSet.StringP(crcConfig.PullSecretFile, "p", "", fmt.Sprintf("File path of image pull secret (download from %s)", constants.CrcLandingPageURL))
 	flagSet.UintP(crcConfig.CPUs, "c", constants.GetDefaultCPUs(crcConfig.GetPreset(config)), "Number of CPU cores to allocate to the instance")
 	flagSet.UintP(crcConfig.Memory, "m", constants.GetDefaultMemory(crcConfig.GetPreset(config)), "MiB of memory to allocate to the instance")
-	flagSet.UintP(crcConfig.DiskSize, "d", constants.DefaultDiskSize, "Total size in GiB of the disk used by the instance")
+	flagSet.Uint64P(crcConfig.DiskSize, "d", constants.DefaultDiskSize, "Total size in GiB of the disk used by the instance")
```

Comment:
I'd prefer to keep DiskSize as uint as long as it's holding GiBs. When we convert it to bytes, then it makes sense to use an uint64. I don't understand how all the changes in this commit help to fix unspecified "integer overflow issue".

Reply:
Done, switched back to `uint`